index.html

```diff
@@ -9,14 +9,12 @@
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1>
+  <h3 class="title">Front-End Tools</h3>
   <div class="tech">
-    <p class="text" id="titleTech">ReactJS </p>
-    <p class="text" id="bold"> Release: 2013</p>
-    <p class="text">React is a declarative,</p>
-    <p class="text">component-based</p>
-    <p class="text">JavaScript library for</p>
-    <p class="text">building user intefaces.</p>
+    <img class="img" src="./assets/react_icon.svg">
```

Comment:
And the `alt` attribute? And change this class name for something more descriptive